core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -236,6 +243,16 @@ private FunctionResolver hour() {
     );
   }
 
+  private FunctionResolver makedate() {
+    return define(BuiltinFunctionName.MAKEDATE.getName(),
+        impl(nullMissingHandling(DateTimeFunction::exprMakeDate), DATE, DOUBLE, DOUBLE));
+  }
+
+  private FunctionResolver maketime() {
+    return define(BuiltinFunctionName.MAKETIME.getName(),
+        impl(nullMissingHandling(DateTimeFunction::exprMakeTime), TIME, DOUBLE, DOUBLE, DOUBLE));
```

Comment:
Why DOUBLE (for hour/minutes), I don't know what maketime(1.5, 1.5, 1.3456) means (at least for hour and minutes)

Reply:
These are rounded?